lutris/services/egs.py

```diff
@@ -150,7 +150,7 @@ class EpicGamesStoreService(OnlineService):
     cookies_path = os.path.join(settings.CACHE_DIR, ".egs.auth")
     token_path = os.path.join(settings.CACHE_DIR, ".egs.token")
     cache_path = os.path.join(settings.CACHE_DIR, "egs-library.json")
-    login_url = "https://www.epicgames.com/id/login?redirectUrl=https://www.epicgames.com/id/api/redirect"
+    login_url = "https://legendary.gl/epiclogin"
```

Comment:
Is there a way to do this _without_ relying on a third-party server?

Reply:
yep fixed